src/components/Form/Accordion/Accordion.scss

```diff
@@ -264,7 +264,7 @@ $breakpoints: (
     }
 
     .right {
-      background-color: $color-gray-light;
+      background-color: $eapp-pink-F39C12;
```

Comment:
Where did this color definition come from? I don't see it mentioned in the JIRA ticket and am hesitant to introduce a one-off color that doesn't have specific instruction.

In the ticket, it says to use `$color-gray`.

Reply:
just wait to see if there are other additional comments so that they can be addressed "simultaneously".

Reply:
revisited and code changes have been.